### ISSUE TITLE
识别实体

### ISSUE THREAD

**Issue by sdutgt** [taishan1994/pytorch_triple_extraction#14]
<img width="499" alt="image" src="https://user-images.githubusercontent.com/93463681/194988498-f4ab87f4-dfae-4526-b148-6d835b034e6e.png">
想问一下，地址都对，为什么还是不可以运行呢？

**Comment by taishan1994** [comment]
> <img alt="image" width="499" src="https://user-images.githubusercontent.com/93463681/194988498-f4ab87f4-dfae-4526-b148-6d835b034e6e.png">
> 
> 想问一下，地址都对，为什么还是不可以运行呢？

你把后面的参数都变成和main.py一行看看。

**Comment by sdutgt** [comment]
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/93463681/195219513-8ba9158a-cf10-4963-baf5-122568e23db9.png">
放在一行之后就显示成这样了，是mac不能用这种代码吗？

**Comment by taishan1994** [comment]
> <img alt="image" width="1079" src="https://user-images.githubusercontent.com/93463681/195219513-8ba9158a-cf10-4963-baf5-122568e23db9.png">
> 
> 放在一行之后就显示成这样了，是mac不能用这种代码吗？

python main.py --bert_dir="../model_hub/chinese-bert-wwm-ext/" --data_dir="../data/dgre/" --log_dir="./logs/" --output_dir="./checkpoints/" --num_tags=5 --seed=123 --gpu_ids="0" --max_seq_len=512 --lr=3e-5 --crf_lr=3e-2 --other_lr=3e-4 --train_batch_size=8 --train_epochs=5 --eval_batch_size=8 --max_grad_norm=1 --warmup_proportion=0.1 --adam_epsilon=1e-8 --weight_decay=0.01 --lstm_hidden=128 --num_layers=1 --use_lstm='False' --use_crf='True' --dropout_prob=0.3 --dropout=0.3

**Comment by sdutgt** [comment]
<img width="914" alt="image" src="https://user-images.githubusercontent.com/93463681/195220636-d60d3f18-e22c-4495-9d25-7e929228ace2.png">
在predict函数中，logic，output这两个地方，显示报错，是不需要管吗？

**Comment by taishan1994** [comment]
> <img alt="image" width="914" src="https://user-images.githubusercontent.com/93463681/195220636-d60d3f18-e22c-4495-9d25-7e929228ace2.png">
> 
> 在predict函数中，logic，output这两个地方，显示报错，是不需要管吗？

修改了下。

**Comment by sdutgt** [comment]
<img width="540" alt="image" src="https://user-images.githubusercontent.com/93463681/195383725-101ea860-bf65-49ad-b436-d6222f601622.png">
重新检查了下，output还是存在没有定义的情况

**Comment by taishan1994** [comment]
> <img alt="image" width="540" src="https://user-images.githubusercontent.com/93463681/195383725-101ea860-bf65-49ad-b436-d6222f601622.png">
> 
> 重新检查了下，output还是存在没有定义的情况,难道是将output换成logits吗

是的，改了。
